insert_data_to_vector_db_weaviate/README.md

```diff
@@ -0,0 +1,44 @@
+# Insert a batch of data to Weaviate vector search engine from a Google Sheet
+
+## Use case
+Insert a batch of data with embeeding vectors to Weaviate vector search engine that is retreived from a Google sheet. Embedding vectors are retreived using the GPT3 models from OpenAI. 
```

Comment:
shouldn't this be weaviate vector database? also, let's maintain consistency with the other sample with capitalization etc.

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/60/commits/86ce07f31841e9fc1ba71fb8d668c85b3864daed.